File: account-management-service/src/foam.py
import foam
from config import get_foam_config

config = get_foam_config()
foam.init(
    service_name="account-management-service",
    api_key=config.foam_api_key,
    is_production=config.is_production,
    enable_traces=True,
    enable_metrics=True
)

File: merchant-integration-api/src/foam.kt
package com.affirm.merchantapi

import com.foam.kotlin.*

object FoamInit {
    fun initialize() {
        Foam.init(
            FoamConfig(
                serviceName = "merchant-integration-api",
                apiKey = System.getenv("FOAM_API_KEY") ?: "",
                isProduction = System.getenv("SPRING_PROFILES_ACTIVE") == "production",
                enableTraces = true,
                enableMetrics = true
            )
        )
    }
}